doc/getting-started.rst

```diff
@@ -0,0 +1,596 @@
+=============================
+Getting started with Buildout
+=============================
+
+.. contents::
+
+.. note::
+
+   In the Buildout documentation, we'll use the word
+   *buildout* to refer to:
+
+   - The Buildout software
+
+     We'll capitalize the word when we do this.
+
+   - A particular application of Buildout, which has a buildout
+     configuration.
```

Comment:
Except I specifically say "application of".  I also say "which has a buildout configuration". Hm.

How about: 

A particular use of Buildout, a directory having a Buildout configuration file.